### ISSUE TITLE
Utilize Cycle learning rate for training

### ISSUE THREAD

**Issue by dmtrs** [wekaco/samplernn-pytorch#25]
https://pytorch.org/docs/1.2.0/optim.html?highlight=cycle%20learning%20rate

Now we support this with pytorch 1.5
